tools/ci_build/github/linux/docker/Dockerfile.manylinux2010_gpu

```diff
@@ -63,6 +66,36 @@ RUN CUDNN_DOWNLOAD_SUM=7eaec8039a2c30ab0bc758d303588767693def6bf49b22485a2c00bf2
     ldconfig
 
 
+# -- training dependencies --
+
+# enable training dependencies by setting build argument ENABLE_TRAINING to a non-empty value
+ARG ENABLE_TRAINING
+
+# install MPI 4.0.0
+RUN test -z $ENABLE_TRAINING || ( \
```

Comment:
You may remove the "test -z $ENABLE_TRAINING" if you want. I think both of them can reuse the same build image.